docs/process-services-cloud/components/form-cloud.component.md

```diff
@@ -160,6 +163,49 @@ For an existing Task both the form and its values will be fetched and displayed.
 
 In this case, only the form definition will be fetched.
 
+#### Enabling fullscreen display for the form
+
+Provide a `displayModeConfiguration` array object containing the fullscreen configuration. You can use the configuration provided in the [`CloudFormRenderingService`](../../../lib/process-services-cloud/src/lib/form/components/cloud-form-rendering.service.ts) as a static member `CloudFormRenderingService.IMPLEMENTED_DISPLAY_MODE_CONFIGURATIONS`, or configure your own if you want to customise the options for the fullscreen display mode.
+
+**MyView.component.html**
+
+```html
+<button (click)="displayFormInFullscreenMode()">
+
+<adf-cloud-form #adfCloudForm
+    [appName]="appName"
+    [taskId]="selectedTask?.id"
+    [showTitle]="false"
+    [showRefreshButton]="false"
+    [showValidationIcon]="false"
+    [displayModeConfigurations]="displayConfigurations">
+</adf-cloud-form>
+```
+
+**MyView.component.ts**
+
+```ts
+import { ViewChild } from '@angular/core';
+import { CloudFormRenderingService, FormCloudComponent } from '@alfresco/adf-process-services-cloud';
+
+export class MyView {
+
+    @ViewChild('adfCloudForm', { static: true })
+    adfCloudForm: FormCloudComponent;
+
+    get displayConfigurations() {
```

Comment:
for the example to showcase full screen, this can be omitted I believe

Reply:
I don't think so, because by default, the full screen display configuration is not provided, we provide just the inline one (the default we were using). 

If you want to be able to use the full screen mode, you need to include a configuration that provides it, for example, the one exported as static member in the `CloudFormRenderingService`. 

See the initial paragraph of this section in the MD file:

> Provide a `displayModeConfiguration` array object containing the fullscreen configuration. You can use the configuration provided in the [`CloudFormRenderingService`](../../../lib/process-services-cloud/src/lib/form/components/cloud-form-rendering.service.ts) as a static member `CloudFormRenderingService.IMPLEMENTED_DISPLAY_MODE_CONFIGURATIONS`, or configure your own if you want to customise the options for the fullscreen display mode.